source/jormungandr/jormungandr/parking_space_availability/bss/tests/atos_test.py

```diff
@@ -78,24 +78,40 @@ def parking_space_availability_atos_support_poi_test():
 
 def parking_space_availability_atos_get_informations_test():
     """
-    Atos validate return good stands informations or None if an error occured
+    Atos validate return good stands informations or or stands with status='Unavailable' if an error occured
```

Comment:
I know this was already like this but maybe change the beginning of this sentence that doesn't really make sense...